ansible/vmware-create-template.yml

```diff
@@ -32,7 +32,7 @@
         name: vmware-template
       vars:
         stage: migrate
-        vcenter_datastore_name: "NFS-Datastore"
+        vcenter_datastore_name: "{{ vcenter_datastore_name }}"
```

Comment:
You can remove this line, since `vcenter_datastore_name` is provided as extra variable.